src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,12 +2,24 @@
 
 ## Unreleased
 
+## 1.6.0
+
+Released 2023-Sep-05
+
 * Increased the character limit of the Meter instrument name from 63 to 255.
   ([#4774](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4774))
 
 * Update default size for `SimpleExemplarReservoir` to `1`.
   ([#4803](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4803))
 
+* Update Metrics SDK to override the default histogram bounds for a set of
+  well-known histogram metrics from AspNetCore and Http runtime and their
+  corresponding instrumentation libraries. These histogram metrics which have
+  their `Unit` as `s` (second) will have their default histogram buckets as `[
+  0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10
+  ]`.
+  ([#4820](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4820))
```

Comment:
nit: Could update the PR separately to include the list of metric names in description.

Reply:
Yes. I commented about the same in the original PR: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4820#discussion_r1316475276